mobile/src/components/ButtonGroup.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { useState } from 'react';
+import { Text, StyleSheet, View, TouchableOpacity } from 'react-native';
+import { Colors } from '../styles/Colors';
+import { TextStyles } from '../styles/TextStyles';
+
+// button labels and callback function for passing key of active button to parent
+type ButtonGroupProps = { btn1: string, btn2: string, btn3: string, onBtnChange };
+
+/**
+ * Renders an inline group of three button tabs
+ */
+export const ButtonGroup: React.FC<ButtonGroupProps> = ({ btn1, btn2, btn3, onBtnChange }) => {
+  const [activeButton, setActiveButton] = useState('btn-1');
+  const buttons = [
+    {
+      key: 'btn-1',
+      label: btn1,
+    },
+    {
+      key: 'btn-2',
+      label: btn2,
+    },
+    {
+      key: 'btn-3',
+      label: btn3,
+    },
+  ];
+
+  return (
+    <View style={{ flexDirection: 'row' }}>
+      {buttons.map(res => {
+        return (
+          <View key={res.key}>
+            <TouchableOpacity
+              style={[styles.button, activeButton === res.key ? styles.buttonActive : styles.buttonInactive]}
+              onPress={() => {
+                setActiveButton(res.key);
+                onBtnChange(res.key);
+              }}
+            >
+              <Text style={[activeButton === res.key ? styles.buttonTextActive : styles.buttonTextInactive, TextStyles.caption2]}>
+                {res.label}
+              </Text>
+            </TouchableOpacity>
+          </View>
+        );
+      })}
+    </View>
+  );
+};
+
+const styles = StyleSheet.create({
+  button: {
+    justifyContent: 'center',
+    width: 74,
+    height: 26,
+    borderWidth: 1,
+    borderRadius: 5,
+    borderColor: Colors.orange,
+    shadowColor: 'black',
```

Comment:
Let's maybe add all of the colors to Colors.ts, even when they're standard colors like #000000 or #FFFFFF. I would call this `Colors.shadowColor` so we could consistently change the style of all of the shadows across the app if we ever wanted to.